src/Types.ts

```diff
@@ -431,3 +433,10 @@ export interface DocumentInfo {
     page_count: number
     size_mb: number
 }
+export interface Range {
+    from?: string
+    to?: string
+    interval: string
+}
```

Comment:
there is already `Terms` interface that includes these fields, along with `Interval` type, no need to cast in `getClosestInterval` function